add using UTMs in product design (#2245)
sourcegraph/handbook: content/departments/product-engineering/product/design/metrics/defining-metrics.md

- If a large number of events are being collected, consider creating a Metrics page in Figma ([example](https://www.figma.com/file/LfQUGTLvfYWxyuFoyQsptB/?node-id=251%3A7408)) to identify the locations and interactions where events will be collected.
- Metrics review: consider requesting a review of your metrics definitions from the analytics team before the project goes to development.

## Tracking events across domains or products

> Often we need to understand how our designs perform when they send traffic to Sourcegraph websites outside of the product we are working on. When these events occur and they end in a visit to a publicly facing Sourcegraph website, we can use Google Analytics [UTM parameters](https://en.wikipedia.org/wiki/UTM_parameters) to track the source of the event.

Reviewer: ```suggestion
Often we need to understand how our designs perform when they send traffic to Sourcegraph websites outside of the product we are working on. When these events occur and end at a publicly facing Sourcegraph website, we can use Google Analytics [UTM parameters](https://en.wikipedia.org/wiki/UTM_parameters) to track the source of the event.
```

Examples of scenarios you may need to track:

- Sign-up links from an IDE extension to Sourcegraph.com
- Links from Sourcegraph.com to a lead generation form on about.sourcegraph.com
- Links from Server to Sourcegraph.com

Examples of events you cannot track with UTM parameters:

- Links from an IDE extension to a Server installation of Sourcegraph (use Pings to track this)
- Links from Sourcegraph.com to external properties like Youtube videos (use event logs to track the outbound click)

How Sourcegraph uses UTM parameters:

- utm_campaign={STRING} A string describing a series of content you are using to promote the link
- utm_medium={STRING} Typically 'in-product', potentially with a product designator, such as in-product-vscide
- utm_source={STRING} Which property sent the traffic. E.g. Sourcegraph.com, VSCIDE, etc.
- utm_content={STRING} A descriptor for the content used in the link

> This can all be a little daunting, and it's hard to know how to name your values for these parameters. The [UTM tracking spreadsheet](https://docs.google.com/spreadsheets/d/1U0HRC5WVz3tsP6z9pqDLG8igTMSf2-pQGhbRoVn_iu0/edit#gid=0) will help you create the link and allow review existing in-product links to use as a guide.

Reviewer: Could we add some additional directions after the step of using the UTM tracking spreadsheet to name your values? These parameters are nominal until an engineer enables(?) the new parameter? (i.e. the work that Philipp is doing in[ this PR](https://github.com/sourcegraph/sourcegraph/pull/30890#issuecomment-1034337118). It's something I'm still not sure of myself even as I'm doing this. :)

Author: DONE. I also added a UTM card to the product tools Figma file.

Author: NOTE: a few of the utm_parameter values have changed due to feedback from marketing. Please update your PRs accordingly.
